modules/core/src/main/java/org/apache/ignite/internal/processors/performancestatistics/FilePerformanceStatisticsReader.java

```diff
@@ -457,6 +462,48 @@ else if (opType == PAGES_WRITE_THROTTLE) {
 
             return true;
         }
+        else if (opType == PME) {
```

Comment:
Would you mind to rewrite 'if' to 'switch' ?

Reply:
Thanks, done.